command/commands/user.py

```diff
@@ -0,0 +1,89 @@
+"""Command parsing for user events"""
+import argparse
+import shlex
+
+
+class UserCommand:
+
+    __command_name = "user"
+    __help = "User Command Reference:\n\n @rocket user\n\n Options:\n\n edit \n --name NAME\n" \
```

Comment:
A small thing (that doesn't have to be changed in this PR, though we should make an issue): eventually we will probably want this to be put into some kind of config file (probably specific to the UserCommand module?) rather than have it hardcoded.